internal/connect/api.go

```diff
@@ -197,6 +236,16 @@ func makeSysInfoBody(distroTarget, namespace string, instanceData []byte) ([]byt
 	payload.InstanceData = string(instanceData)
 	payload.Namespace = namespace
 
+	if includeUptimeLog {
+		uptimeLog, err := readUptimeLogFile(UptimeLogFilePath)
```

Comment:
From the function implementation, it's pretty much guaranteed that if err == nil, then uptimeLog != nil, right? Then this can be further simplified to something like:

```go
if uptimeLog, err := readUptimeLogFile(UptimeLogFilePath); err != nil {
	util.Info.Print("Unable to system uptime log")
} else {
	payload.OnlineAt = uptimeLog
}
```

Reply:
It might also be a good idea to include the error message as a debug line. That is, we make an effort inside of `readUptimeLogFile` to get the error and the caller does not use it at all 😄

Reply:
Done